### ISSUE TITLE
BY-23  As a student with accessibility issues, I want to find the shortest path from point A to point B which take into account my preferences. 

### ISSUE THREAD

**Issue by JenYn** [jaychenblue/scout-concordia#23]
Multiple paths are displayed and the user can choose the one they want. The most accessible shortest path is displayed when a user with accessibility preferences inputs a starting point and a destination.
Related to issue #11

**Comment by JenYn** [comment]
The settings must be read from the local storage file. Based on the settings, the path will make use of elevators, ramps, etc. 

**Acceptance criteria:**
- [ ] Elevator is used if the user must change floors
- [ ] ETA is accurate and is adjusted according to disability preferences (must be longer than regular ETA for same path with accessibiltiy settings turned off)

**Comment by JenYn** [comment]
We will have to re-evaluate the estimate for this feature as we need to understand how to take accessibility amenities into account when creating the path.

**Comment by vincentcerri** [comment]
**Implementation details**
To solve this issue, we added a Boolean that declares whether or not the application should be using accessibility issues. If it is set to false, the app will find the shortest distance the default way.  Refer to #22. The app will navigate floor by floor using the escalators. If the value is set to false however, rather than searching for an escalator when going from floor to floor, the path will be directed to the nearest elevator. refer to issue #30
